book/src/tutorial/generate_docs.md

```diff
@@ -0,0 +1,36 @@
+# Generating documentation
+And finally the last feature! Just run the following command in the folder of the safe wrapper crate:
```

Comment:
Just to be very clear, usually we don't embed the docs into the library itself for licensing reasons, so I would make it very clear that you have to do this step only on the CI used to publish the docs and mention you can use the helper script with `python3 ./generate.py --embed-docs` or `python3 ./generate.py --strip-docs` for reverting those changes.

Additionally, I feel like this chapter would be the perfect place to mention the need for making sure the `dox` feature is properly set on your `Cargo.toml` for your safe rust wrappers and that the `dox` feature avoids linkings against the library so that building the docs don't fail on docs.rs

Reply:
Is there somewhere I can read up on these licensing issues so that I can explain them? I don't know a lot about licensing and don't understand why  for example it would be a problem to generate a wrapper with docs for a MIT licensed library that depends on GTK4 libraries (LGPL) and their wrapper (MIT).

I added d6032f674f419b6f8326a636ee7347561d6f8b7d to address your suggestions regarding the dox feature. There already is a description in the chapter about the Cargo.toml file of the safe wrapper, but your suggestion to also add it to the chapter about the docs makes sense in case somebody skips a chapter for example.